debian/control

```diff
@@ -18,6 +18,7 @@ Build-Depends: debhelper-compat (= 13),
                python3-samba,
                samba,
                samba-dsdb-modules,
+               polkitd-pkla,
```

Comment:
This is in universe, we should either MIR it or downgrade it to suggests until we support the "new" format.